package.json

```diff
@@ -12,7 +12,7 @@
     "start": "node ./streaming/index.js",
     "test": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:jest",
     "test:lint": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:lint:sass",
-    "test:lint:js": "eslint --ext=js . --cache --report-unused-disable-directives",
+    "test:lint:js": "eslint --ext=.js,.jsx . --cache --report-unused-disable-directives",
```

Comment:
```suggestion
    "test:lint:js": "eslint . --cache --report-unused-disable-directives",
```

Reply:
When doing this, the `.jsx` files are no longer processed by `eslint` 🤔

Reply:
No problem, I couldn't recall if it defaulted to both extensions, since I thought it picked up other things like Typescript files with the plain `.` descovery